### PR TITLE
Eth2 used in the deployment

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

Please provide a description and a link to the underlying ticket

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: testnet/testnet-local-eth2network.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+fi
+
+# start the geth network
+echo "Starting the gethnetwork.."
+docker network create --driver bridge node_network || true
+docker run --name=eth2network -d \
+  --network=node_network \
+  -p 8025:8025 -p 8026:8026 -p 9000:9000 -p 9001:9001 \
+  --entrypoint /home/obscuro/go-obscuro/integration/eth2network/main/main \
+   testnetobscuronet.azurecr.io/obscuronet/eth2network:latest \
```
Comment (StefanIliev545) [inline testnet/testnet-local-eth2network.sh:62]: Should we have dev_ here for dev_testnet? Given that we have some go code orchestrating some bits it might be good to have the separation to prevent accidental overwrites.

File: integration/eth2network/eth2_configs.go [hunk 2]
```diff
@@ -40,7 +44,7 @@ ALTAIR_FORK_VERSION: 0x20000090
 # Merge
 BELLATRIX_FORK_EPOCH: 4
 BELLATRIX_FORK_VERSION: 0x20000091
-TERMINAL_TOTAL_DIFFICULTY: 2
+TERMINAL_TOTAL_DIFFICULTY: 10
```
Comment (StefanIliev545) [inline integration/eth2network/eth2_configs.go:47]: Why the change?

**Review by StefanIliev545 — APPROVED** [review]
Just a question and optional change if you feel its appropriate

File: testnet/testnet-local-eth2network.sh (hunk 1 above)
Comment (otherview) [reply testnet/testnet-local-eth2network.sh:62]: This is only used for local testnet. 
I think it's time we start spliting the scripts for local usage into a diff folder.
Opened a ticket for it https://github.com/obscuronet/obscuro-internal/issues/1413

File: integration/eth2network/eth2_configs.go (hunk 2 above)
Comment (otherview) [reply integration/eth2network/eth2_configs.go:47]: It was reaching the difficulty too fast and seem'd to get the network in a stale state.
